webapi/src/main/scala/org/knora/webapi/responders/admin/GroupsResponderADM.scala

```diff
@@ -886,10 +880,6 @@ class GroupsResponderADM(responderData: ResponderData)
     apiRequestID: UUID
   ): Future[GroupOperationResponseADM] =
     if (changedGroup.status) {
-      // group active. no need to remove members.
-      log.debug("removeGroupMembersIfNecessary - group active. no need to remove members.")
-      FastFuture.successful(GroupOperationResponseADM(changedGroup))
-    } else {
       // group deactivated. need to remove members.
       log.debug("removeGroupMembersIfNecessary - group deactivated. need to remove members.")
       for {
```

Comment:
didn't you now invert the logic of this if statement?

Reply:
Yes, but I forgot about exclamation mark. But now I'm not sure anymore if this will work as we expect.